buildenv/jenkins/jobs/infrastructure/copyrightCheck

```diff
@@ -64,17 +64,34 @@ timeout(time: 6, unit: 'HOURS') {
                         sh 'exit 1'
                     }
 
+                    if (fileExists("${WORKSPACE}/.copyright_whitelist")) {
+                        WHITE_LIST = readFile '.copyright_whitelist'
+                        WHITE_LIST = WHITE_LIST.split("\\r?\\n")
+                    } else {
+                        WHITE_LIST = ""
+                    }
+
                     FILES_LIST.each() {
                         println "Checking file: '${it}'"
-                        RESULT = sh (
-                            script: "grep -qE ${REGEX} '${it}'",
-                            returnStatus: true)
-                        if(RESULT != 0) {
-                            echo "FAILURE - Copyright date in file: '${it}' appears to be incorrect"
-                            FAIL = true
-                            BAD_FILES << "${it}"
+                        Boolean IGNORE = false
+                        WHITE_LIST.each() { filename ->
```

Comment:
Are you able to use the `contains()` method here rather than writing your own loop?